submissions/AndriiZahumnyi/js-dom/script/index.js

```diff
@@ -0,0 +1,33 @@
+
+/* Toggle menu */
+
+const hamburgerToggle = document.querySelector('#menu-toggle');
+
+hamburgerToggle.addEventListener('click', function (e) {
+  
+  e.preventDefault();
+
+  document.querySelector('.flexmenu').classList.toggle('active');
+});
+
+/* Aside menu -  interactive side-menu*/
```

Comment:
redundant comment. 
I'd recommend you watching this [video](https://youtu.be/yhF7OmuIILc) about the comments